tests/unit/Params/UserParamsTest.php

```diff
@@ -33,9 +33,12 @@ public function testOrcid(): void
         $orcid = '1234-5678-1212-0001';
         $params = new UserParams('orcid', $orcid);
         $this->assertEquals($orcid, $params->getContent());
+        // Remove orcid
```

Comment:
I meant do the actual patch and check that orcid is null after setting it. It's not really interesting to test the UserParams object. What you want to test is the full action.

Reply:
oups my bad.
I moved it to the UsersTest, as it is testing the update of the user and not the param itself indeed